UndertaleModTool/MainWindow.xaml.cs

```diff
@@ -1730,7 +1734,7 @@ private void CopyItemName(object obj)
 
             if (obj is UndertaleNamedResource namedRes)
                 name = namedRes.Name?.Content;
-            else if (obj is UndertaleString str)
+            else if (obj is UndertaleString str && str.Content?.Length > 0)
```

Comment:
Why check for the length here, if title converter already does it?

Reply:
So it wouldn't copy "(empty string)", as it's useless.